### PR TITLE
archetypes added

### PR DESCRIPTION
**Pull request by a-chappelear**
## Changes proposed in this pull request:

- Archetype method card added, persona removed, navigation updated. Links updated on research page.
-
-

## security considerations

[Note the any security considerations here, or make note of why there are none]

### CODE REVIEW

**Comment by a-chappelear** [comment]
Ticket - #289 ; the only link I found referring to the persona method was on https://guides.18f.gov/ux-guide/research/make-research-actionable/

https://guides.18f.gov/ux-guide/research/plan/, this mentions personas but it’s linking to a blog post (not the method card)

File: content/methods/decide/archetypes.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+layout: layouts/method-card
+tags: methods
+eleventyNavigation:
+  key: methods
+  parent: Decide
+  title: Archetypes
+method:
+  title: Archetypes
+  what: Archetypes are visual documents that represent a group of users based on their shared goals, needs, attitudes, behaviors, and/or pain points.
+  why: Unlike demographic data that can explicitly introduce creative blocks or bias, archetypes can help better understand product or service users who share the same core desires.
```
Comment (MelissaBraxton) [inline content/methods/decide/archetypes.md:16]: Replace the word "explicitly" with "unintentionally"

Comment (MelissaBraxton) [inline content/methods/decide/archetypes.md:16]: Include "behaviors" in the last sentence: "...users who share the same behaviors and core desires."

**Review by MelissaBraxton — CHANGES_REQUESTED** [review]
A few tiny word additions. Otherwise, looks good to me!

**Comment by a-chappelear** [comment]
Updated w/ edits!

**Comment by a-chappelear** [comment]
> @a-chappelear Looking good! I hate to say it, but we need to remove the NN link too, since they offer training for $. After that, I think we're good to go!

updated!

**Review by MelissaBraxton — APPROVED** [review]
LGTM!

**Pull request merged**